src/pages/userPage/userPage.tsx

```diff
@@ -69,43 +69,50 @@ const UserPage = () => {
     const [tag, setTag] = useState<Tag>(Tag.Posts)
     const [sort, setSort] = useState<QueryType>(QueryType.Boost)
     const [isDropdown, setIsDropdown] = useState<boolean>(false)
-    const myPosts =
-        postContext.feedsByQuery[
-            postContext.feedKey(sort, user.currentEpochKeys)
-        ] || []
-    const myComments =
-        postContext.commentsByQuery[
-            postContext.feedKey(sort, user.currentEpochKeys)
-        ] || []
+    const [myPosts, setMyPosts] = useState<string[]>([])
```

Comment:
well, I think we don't have to make the feedkey be like `{query}-{epk1}_{epk2}...` since we have to show all posts related to the user instead of only some epks.